### DISCUSSION TITLE
Possible to override parts of a theme?

### DISCUSSION THREAD

**Discussion by benfrain** [nvim-lualine/lualine.nvim#996] [Q&A]
If I have a theme, and I would like to overide the colors just for the mode, for example, is it possible to do that? 

For example if I have the theme set to 'auto', but want the following for the command mode colors:

```
command = {
    a = { bg = theme.vcs.removed, fg = theme.ui.bg },
    b = { bg = theme.ui.bg, fg = theme.vcs.removed },
},
```

Is it possible to do that through the setup functIon?

**Comment by IRooc** [comment]
I'm also wondering this.
